lib/blacklight/configuration.rb

```diff
@@ -402,5 +401,22 @@ def add_action(config_hash, name, opts)
       yield(config) if block_given?
       config_hash[name] = config
     end
+
+    def deep_transform_values_in_object(object, &block)
+      return object.deep_transform_values(&block) if object.respond_to?(:deep_transform_values)
+      _deep_transform_values_in_object(object, &block)
+    end
+
+    def _deep_transform_values_in_object(object, &block)
```

Comment:
Can you drop a comment here about being able to remove this when we drop Rails 5 support?